ci/py3.8.yml

```diff
@@ -7,11 +7,12 @@ dependencies:
   - apache-beam
   - black
   - boto3
-  - cfgrib<0.9.9.0
+  - cfgrib>=0.9.10.2 # Needed for grib2 reference recipe to work
```

Comment:
Does this mean that we need to update the requirements in `setup.cfg` to include `cfgrib` and `eccodes`?

Reply:
@andersy005, just updated.